demisto_sdk/commands/validate/validators/base_validator.py

```diff
@@ -138,7 +138,9 @@ def should_run(
                     self.related_file_type,
                 ),
                 not is_support_level_support_validation(
-                    self.error_code, support_level_dict, content_item.support_level
+                    self.error_code,
+                    support_level_dict,
+                    content_item.support,  # type: ignore
```

Comment:
why ignoring type?

Reply:
since we delletted these lines in BaseContent:
    @property
    def support_level(self) -> str:
        raise NotImplementedError

    and I think its better to ignore than to addd a property that reises and never in use

Reply:
isn't it because `content_item.support` is `Optional[str]` and not `str`?
Can we make sure it's `str`, as we expect it to be in reality after the validator? It requires Pack's `support` to be `str` as well

Reply:
there's always a way to make mypy happy. two things we can try:
1. not sure it will work but I have a feeling that yes:
```
if isinstance(integration_script, IntegrationScript):
        file = integration_script.path.parent / f"{integration_script.path.stem}.py"
        if not file.exists():
            return results
    xsoar_linter_env = build_xsoar_linter_env_var(integration_script)
    env.update(xsoar_linter_env)
    command = build_xsoar_linter_command(integration_script.support)
else:
   return results
```

2. if the one above doesn't work:
```
from typing import cast
...
integration_script = cast(IntegrationScript, integration_script)
```

Reply:
we're getting:
```
"ContentTypes" has no attribute "support"  [attr-defined]
```
mypy is asking you legitimately, is there any chance `content_item` will not have this property?
if there is a chance, make sure it doesn't reach this code block!
if not - why is it "BaseContent" and not one of the actual type that really has this field?

in both cases, this can be solved by putting the code in the `if isinstance(content_item, ContentItem)` block, or `content_item = cast(ContentItem, content_item)` (I prefer the first one)